media/css/pocket/components/add.scss

```diff
@@ -48,9 +50,8 @@ $image-path: '/media/img/pocket';
         &:hover,
         &:focus,
         &:active {
-            text-decoration: underline !important;
```

Comment:
Friendly FYI: the reason I made it have an underline on hover was because it's what I saw on the [new homepage design](https://getpocket.com/en/) some months back. I'll ask designers if this was intentional, just to make sure if this is what they want for pseudos

Reply:
Thanks for the info Reem. This is probably something we should consider in relation to improving alignment with Protocol for the future. It's also a bit odd that many buttons on the pocket website already don't do this, with the exception of the red buttons here.

Reply:
Last-minute CSS on my part 🙈 FWIW, I know it's in use already but I'm not convinced the lighter/darker color shade is sufficient feedback to pass [WCAG 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=141#use-of-color). I'll mention to the designers, but aligning more with Protocol is a good point too.

Reply:
If the current shade doesn’t pass it should be pretty easy to tweak, thanks for the fallout @maureenlholland

Reply:
Er… call-out 😂

Reply:
The shade is probably fine @alexgibson appreciate you sorting out the `!important` debt! My concern is that, as feedback for the user that this element is interactive and you are hovering over it, we are only using a color change and that might not be enough feedback without the addition of total color inversion or underline/outline/border or size change.

Reply:
Yeah this is really a much bigger can of worms, as 99% of all our buttons in bedrock (and protocol) already do this. I'm all for making a change if that improves things for user's experiences. But I'd strongly favour doing so at a component level, and not on a page by page basis.